modes/longitudinal/src/index.ts

```diff
@@ -192,7 +177,7 @@ function modeFactory({ modeConfiguration }) {
             id: ohif.layout,
             props: {
               leftPanels: [tracked.thumbnailList],
-              rightPanels: [dicomSeg.panel, tracked.measurements],
+              rightPanels: [cornerstone.segmentation, tracked.measurements],
```

Comment:
Can the order be swapped here?  It seems to me that measurements are more common that segmentations, so starting showing the measurements first would be better.

Reply:
We recently reordered it. I think we should ensure that panels switch their active status if they are not in the active panel context. For example, if the segmentation panel is open and a measurement is added, we should automatically switch to the measurement panel, and vice versa. https://github.com/OHIF/Viewers/issues/4459